README.md

```diff
@@ -36,16 +36,71 @@ With the package installed, we load in the data. To get an idea on how the trans
 import composeml as cp
 
 df = cp.datasets.transactions()
```

Comment:
for consistency with Featuretools can we make this `cp.demos.load_transactions()`?

Reply:
Yes, will update.